src/test/consensus/Validations_test.cpp

```diff
@@ -1080,6 +1080,14 @@ class Validations_test : public beast::unit_test::suite
         }
     }
 
+    void
+    testDifferentCookies()
```

Comment:
I think testing this would be useful; I'm a little rusty on the test harness here but I think it should be feasible and can try to help.

Reply:
Appreciate the offer. I'll work with someone else to try and get these tests so that more people learn about consensus instead of having all the information locked up in the heads of three or four people.

Reply:
I agree that this is important. I'm not sure it needs to be addressed before this is merged (that's up to Brad and Nik), but it would be good to assign this task to someone before this is merged, just so we don't lose track of the task.

Reply:
I will be happy to work on it, probably after the negative UNL network tests, if that is OK.